DESCRIPTION

```diff
@@ -5,9 +5,11 @@ Authors@R: c(
     person("Robert", "Gentleman", role="aut"),
     person("Seth", "Falcon", role="ctb"),
     person("Robert", "Castelo", role="ctb"),
+    person("Dennis", "Ndubi", role ="ctb",
+        comment = "Converted GOstatsHyperG vignette from Sweave to RMarkdown / HTML."),
     person("Bioconductor Package Maintainer",
-        email = "maintainer@bioconductor.org", role="cre"))
-Description: A set of tools for interacting with GO and microarray
+        email = "maintainer@bioconductor.org", role = "cre"))
+Description: A set of t ools for interacting with GO and microarray
```

Comment:
Be consistent regarding spacing around `=`. Since the existing text has no spaces around `=`, we should follow that style. Fix for lines 9 and 11.

Line 12 introduces a spelling error.